tests/analysis/test_ols_analysis.py

```diff
@@ -14,5 +14,5 @@ def test_binary_treatment():
 
 def test_get_pvalue():
     analysis_df_full = pd.concat([analysis_df for _ in range(100)])
-    analyser = OLSAnalysis()
+    analyser = OLSAnalysis(hypothesis="left_tailed")
```

Comment:
I'd add a separate test, and I'd test the functionality itself of the p-value transformer

Reply:
as in, check that it is dividing by 2 or doing the other stuff when necessary